src/main/scala/mesosphere/marathon/MarathonModule.scala

```diff
@@ -340,42 +343,41 @@ class MarathonModule(conf: MarathonConf, http: HttpConf)
   @Singleton
   def provideTaskFailureRepository(
```

Comment:
They are all coming from StorageModule. I'm still working through moving migration into storage module and writing the migration from zk -> zk2.